I see this error

```
[2024-07-11 22:02:40 +0000] [ERROR] [log.py:241] Internal Server Error: /init/ea7cbaeb-ae60-4fca-9317-83e6c38c3a13/
clrt_web         | Traceback (most recent call last):
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
clrt_web         |     response = get_response(request)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
clrt_web         |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
clrt_web         |     return self.dispatch(request, *args, **kwargs)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/utils/decorators.py", line 48, in _wrapper
clrt_web         |     return bound_method(*args, **kwargs)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
clrt_web         |     return view_func(request, *args, **kwargs)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/views/generic/base.py", line 143, in dispatch
clrt_web         |     return handler(request, *args, **kwargs)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/lti_tool/views.py", line 42, in get
clrt_web         |     return self.get_oidc_response(request, registration_uuid, request.GET)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/lti_tool/views.py", line 64, in get_oidc_response
clrt_web         |     ).redirect(redirect_url)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/pylti1p3/oidc_login.py", line 158, in redirect
clrt_web         |     redirect_obj = self._prepare_redirect(launch_url)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/pylti1p3/oidc_login.py", line 141, in _prepare_redirect
clrt_web         |     auth_login_return_url = self._prepare_redirect_url(launch_url)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/pylti1p3/oidc_login.py", line 107, in _prepare_redirect_url
clrt_web         |     self._session_service.save_nonce(nonce)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/pylti1p3/session.py", line 42, in save_nonce
clrt_web         |     self._set_value(self._get_key("nonce", nonce), True)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/pylti1p3/session.py", line 30, in _set_value
clrt_web         |     self.data_storage.set_value(key, value, exp=self._launch_data_lifetime)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/pylti1p3/launch_data_storage/cache.py", line 36, in set_value
clrt_web         |     self._get_cache().set(key, value, exp)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/core/cache/backends/redis.py", line 191, in set
clrt_web         |     self._cache.set(key, value, self.get_backend_timeout(timeout))
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/utils/functional.py", line 47, in __get__
clrt_web         |     res = instance.__dict__[self.name] = self.func(instance)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/core/cache/backends/redis.py", line 172, in _cache
clrt_web         |     return self._class(self._servers, **self._options)
clrt_web         |   File "/usr/local/lib/python3.10/site-packages/django/core/cache/backends/redis.py", line 39, in __init__
clrt_web         |     import redis
clrt_web         | ModuleNotFoundError: No module named 'redis'
```

--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -12,7 +12,7 @@ services:
     ports:
       - "4306:3306"
     volumes:
-      - clrt-mysql-data:/var/lib/mysql
+      - ./.data/mysql:/var/lib/mysql
     container_name: clrt_mysql_host
 
   redis:
@@ -44,5 +44,4 @@ services:
     container_name: clrt_web
 
 volumes:
-  clrt-mysql-data:
   clrt-redis-data:

--- a/requirements.txt
+++ b/requirements.txt
@@ -8,3 +8,4 @@ django-mysql==4.14.0
 gunicorn==22.0.0
 whitenoise==6.7.0
 debugpy==1.8.2
+redis==5.0.7

